frontend/public/components/dashboard/top-consumers-card/strings.js

```diff
@@ -1,6 +1,6 @@
 export const CPU_DESC = 'CPU time';
 export const MEMORY_DESC = 'Memory consumption';
-export const STORAGE_DESC = 'Filesystem IO time';
+export const STORAGE_DESC = 'Average I/O time';
```

Comment:
Should this be Average Filesystem I/O time? I mean I guess that's implied.

Reply:
IMO `Average I/O time` is enough as you need to pick ` By Storage` from dropdowns above the metric first.